CHANGELOG.md

```diff
@@ -0,0 +1,15 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [1.0.9] - 2023-08-17
+
+### Changed
+
+- `formatTimestamp()` function will return time according to timezone of the user.
```

Comment:
Function will return formatted time as per Timezone passed in parameter. Previously IST was the default Timezone being used.

Reply:
@jainarpit014  we are not passing timezone in parameter. It is automatically taking it.